StripeFinancialConnections/StripeFinancialConnections/Source/Native/ManualEntry/ManualEntryFormView.swift

```diff
@@ -207,7 +195,6 @@ extension ManualEntryFormView: ManualEntryTextFieldDelegate {
     }
 
     func manualEntryTextFieldDidBeginEditing(_ textField: ManualEntryTextField) {
```

Comment:
is this still required from the `ManualEntryTextFieldDelegate` ?

Reply:
cleaned this up, thank you!